From f8ef183f06888561281b8ee480181c97bae97bb5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Mon, 22 Jul 2024 18:01:03 +0200
Subject: [PATCH] Remove structure set

---
 pylammpsmpi/wrapper/ase.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/pylammpsmpi/wrapper/ase.py b/pylammpsmpi/wrapper/ase.py
index 0d7ddc7..1886373 100644
--- a/pylammpsmpi/wrapper/ase.py
+++ b/pylammpsmpi/wrapper/ase.py
@@ -145,13 +145,6 @@ def interactive_structure_setter(
         el_eam_lst,
         calc_md=True,
     ):
-        if self._structure is not None:
-            old_symbols = get_species_symbols(structure=self._structure)
-            new_symbols = get_species_symbols(structure)
-            if any(old_symbols != new_symbols):
-                raise ValueError(
-                    f"structure has different chemical symbols than old one: {new_symbols} != {old_symbols}"
-                )
         self.interactive_lib_command(command="clear")
         control_dict = set_selective_dynamics(structure=structure, calc_md=calc_md)
         self.interactive_lib_command(command="units " + units)
